test/expected/version.out

```diff
@@ -5,7 +5,7 @@
 select pg_typeof(git) from _timescaledb_internal.get_git_commit() AS git;
```

Comment:
I suggest changing this to typeof each returned column. Or at least add an additional line to the test to do that, similar to get_os_info below. Will it work for NULL values?